pylint/lint/pylinter.py

```diff
@@ -609,6 +621,38 @@ def register_checker(self, checker):
         if not getattr(checker, "enabled", True):
             self.disable(checker.name)
 
+    def enable_fail_on_messages(self):
+        vals = self.config.fail_on
```

Comment:
```suggestion
        fail_on_msgs= self.config.fail_on
```

Reply:
FYI flipped back on these as the values in `self.config.fail_on` are not necessarily messages, but values that could be category letter, message id, or message symbol.